msg/en_US.utf8/utils.msg

```diff
@@ -1309,3 +1310,25 @@ valid options:\n\
                                   for flashback; default: outputs only SQL statements for flashback\n\
     --oldest                      flashback is performed in chronological order for SQL statements executed from start time to end time;\n\
                                   default: latest\n
+
+$set 59 MSGCAT_UTIL_SET_MEMMON
+1 Mutually exclusive options cannot be used together.\n
+2 'memory_monitorting' option is turned off. Please set system parameter 'memory_monitoring' in 'cubrid.conf'.\n
+3 Argument %1$s in option '-m(--module)' does not match in the list of registered server's module.\n
+4 -c(--tran-count) option must be used with -t(--transaction) option.\n
+59 Memmon cannot run as standalone mode.\n
+60 \
+memmon: Provides memory usage information of a server.\n\
+usage: %1$s memmon [OPTION] database-name\n\
+\n\
+valid options:\n\
+    -m, --module=MODULE_NAME      print the module's total memory usage and memory usage details.\n\
+                 MODULE_INDEX\n\
+                 b(or brief)      print summary of memory usage of the entire registered modules.\n\
+    -t, --transaction             print transaction id and memory usage of the entire active transaction.\n\
```

Comment:
1. print -> display
2. I don't think it is good to describe specific displayed item like transaction id. It is enough to describe the purpose of this option like 'display memory usages of active transactions'. (consider other option too)